src/assets/scripts/client/InputController.js

```diff
@@ -332,104 +268,82 @@ export default class InputController {
         }
     }
 
+    // TODO: This does hardly anything of value. It should probably just be removed to simplify things.
```

Comment:
This comment worries me 😱 

What _is_ this method doing now, isn't this a generic method for any keyboard event?

Reply:
Well there were _two_ keydown handlers, one for when we were selected in the command bar and another when we weren't. We can instead just do the same action regardless, and when complete, return the focus to the command bar.

See 154392c2f1c8a248610863a00502c7869d42ce56 for what I've just now done done to try to reorganize as such.